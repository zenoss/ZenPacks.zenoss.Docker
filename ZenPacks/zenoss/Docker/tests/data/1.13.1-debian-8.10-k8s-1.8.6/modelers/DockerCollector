Docker version 1.13.1, build 092cba3
__SPLIT__
CONTAINER ID                                                       IMAGE                                                                                                                      COMMAND                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            CREATED             STATUS              PORTS               NAMES
4f2e3c742f4f1d54fbe3eb86d378627a96c59bd05c1f8c4f867eb2060be0171f   kope/dns-controller@sha256:97f80ad43ff833b254907a0341c7fe34748e007515004cf0da09727c5442f53b                                "/usr/bin/dns-controller --watch-ingress=false --dns=gossip --gossip-seed=127.0.0.1:3999 --zone=*/* -v=2"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          40 minutes ago      Up 40 minutes                           k8s_dns-controller_dns-controller-5cbcd846f9-r8km7_kube-system_c568bedb-0d13-11e8-b83e-0267931fa8b6_0
91bd7d905685984cb986156636c87c2cc45bf75a2fa4a341e6f876ffae154a68   gcr.io/google_containers/pause-amd64:3.0                                                                                   "/pause"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           41 minutes ago      Up 41 minutes                           k8s_POD_dns-controller-5cbcd846f9-r8km7_kube-system_c568bedb-0d13-11e8-b83e-0267931fa8b6_0
ce56193cdac7a26e9ec80e8099b1033819c50705627a1247672eb64a4f4bc042   gcr.io/google_containers/etcd@sha256:19544a655157fb089b62d4dac02bbd095f82ca245dd5e31dd1684d175b109947                      "/bin/sh -c '/usr/local/bin/etcd 2>&1 | /bin/tee -a /var/log/etcd.log'"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            42 minutes ago      Up 42 minutes                           k8s_etcd-container_etcd-server-events-ip-172-20-54-17.ec2.internal_kube-system_d291a7c4584f864ad771c279f501226b_0
3aa35b7f84789c681444430fb22cf0591b126b610774a84c828f6b632d5ae806   gcr.io/google_containers/kube-proxy@sha256:4dc8aefa2bfe010b5cd820de67e03493a37489696f71339da604de47815ec00a                "/bin/sh -c '/usr/local/bin/kube-proxy --cluster-cidr=100.96.0.0/11 --conntrack-max-per-core=131072 --hostname-override=ip-172-20-54-17.ec2.internal --kubeconfig=/var/lib/kube-proxy/kubeconfig --master=https://127.0.0.1 --oom-score-adj=-998 --resource-container=\"\" --v=2 2>&1 | /usr/bin/tee -a /var/log/kube-proxy.log'"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  42 minutes ago      Up 42 minutes                           k8s_kube-proxy_kube-proxy-ip-172-20-54-17.ec2.internal_kube-system_be7ce701e54caaf75dcc19fcdc7eb5bb_0
ae1b68661a64df089501c1f455eec44c07ae891d5272d02c98fe63bec9a1475f   gcr.io/google_containers/etcd@sha256:19544a655157fb089b62d4dac02bbd095f82ca245dd5e31dd1684d175b109947                      "/bin/sh -c '/usr/local/bin/etcd 2>&1 | /bin/tee -a /var/log/etcd.log'"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            42 minutes ago      Up 42 minutes                           k8s_etcd-container_etcd-server-ip-172-20-54-17.ec2.internal_kube-system_0db13284a988626a3691585593cee6fa_0
c699903676052404e0ba754eb7f43c21e36b6833568bbd0f42191ba4ca8bcb05   gcr.io/google_containers/kube-scheduler@sha256:4dcf7b2872bd9086ce236535bf03d2077252aebedbcad5de324042edd428f478            "/bin/sh -c '/usr/local/bin/kube-scheduler --kubeconfig=/var/lib/kube-scheduler/kubeconfig --leader-elect=true --v=2 2>&1 | /bin/tee -a /var/log/kube-scheduler.log'"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              42 minutes ago      Up 42 minutes                           k8s_kube-scheduler_kube-scheduler-ip-172-20-54-17.ec2.internal_kube-system_cae1661ea6d72df63c1db473df0cfab8_0
cff2a01ebe258d4a77bb2f037ec6a323f5518250e937797df2190bf42ed6d53d   gcr.io/google_containers/kube-apiserver@sha256:f0658f0c3344abaae3217d6e08612ea8816a0cdeea1fdd48776dd55d07f17d30            "/bin/sh -c '/usr/local/bin/kube-apiserver --address=127.0.0.1 --admission-control=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota --allow-privileged=true --anonymous-auth=false --apiserver-count=1 --authorization-mode=AlwaysAllow --basic-auth-file=/srv/kubernetes/basic_auth.csv --client-ca-file=/srv/kubernetes/ca.crt --cloud-provider=aws --etcd-servers-overrides=/events#http://127.0.0.1:4002 --etcd-servers=http://127.0.0.1:4001 --insecure-port=8080 --kubelet-preferred-address-types=InternalIP,Hostname,ExternalIP --proxy-client-cert-file=/srv/kubernetes/apiserver-aggregator.cert --proxy-client-key-file=/srv/kubernetes/apiserver-aggregator.key --requestheader-allowed-names=aggregator --requestheader-client-ca-file=/srv/kubernetes/apiserver-aggregator-ca.cert --requestheader-extra-headers-prefix=X-Remote-Extra- --requestheader-group-headers=X-Remote-Group --requestheader-username-headers=X-Remote-User --secure-port=443 --service-cluster-ip-range=100.64.0.0/13 --storage-backend=etcd2 --tls-cert-file=/srv/kubernetes/server.cert --tls-private-key-file=/srv/kubernetes/server.key --token-auth-file=/srv/kubernetes/known_tokens.csv --v=2 2>&1 | /bin/tee -a /var/log/kube-apiserver.log'"   42 minutes ago      Up 42 minutes                           k8s_kube-apiserver_kube-apiserver-ip-172-20-54-17.ec2.internal_kube-system_94d22b056814f1dd76157ab3970e3bd3_0
9d2919d80e851414e1fb63f395e2b91b7a5a97117122a819d9d25bdcfdd8e77a   gcr.io/google_containers/kube-controller-manager@sha256:f06b0a323511a8d5a0c47c4229c3580390a5232761eade11bbe9356739356514   "/bin/sh -c '/usr/local/bin/kube-controller-manager --allocate-node-cidrs=true --attach-detach-reconcile-sync-period=1m0s --cloud-provider=aws --cluster-cidr=100.96.0.0/11 --cluster-name=cluther-kops.k8s.local --cluster-signing-cert-file=/srv/kubernetes/ca.crt --cluster-signing-key-file=/srv/kubernetes/ca.key --configure-cloud-routes=true --kubeconfig=/var/lib/kube-controller-manager/kubeconfig --leader-elect=true --root-ca-file=/srv/kubernetes/ca.crt --service-account-private-key-file=/srv/kubernetes/server.key --use-service-account-credentials=true --v=2 2>&1 | /bin/tee -a /var/log/kube-controller-manager.log'"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       42 minutes ago      Up 42 minutes                           k8s_kube-controller-manager_kube-controller-manager-ip-172-20-54-17.ec2.internal_kube-system_13b92a993591a89c5e20f9facf926675_0
bc96cba9431214475a6df4449a92cb4fc6597f562e42f1747973881bd7d24c4f   gcr.io/google_containers/pause-amd64:3.0                                                                                   "/pause"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           42 minutes ago      Up 42 minutes                           k8s_POD_etcd-server-events-ip-172-20-54-17.ec2.internal_kube-system_d291a7c4584f864ad771c279f501226b_0
9243cbcb485b8a22b06f728512757109e7f56e664b5e3829f9e2320083480b55   gcr.io/google_containers/pause-amd64:3.0                                                                                   "/pause"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           42 minutes ago      Up 42 minutes                           k8s_POD_kube-proxy-ip-172-20-54-17.ec2.internal_kube-system_be7ce701e54caaf75dcc19fcdc7eb5bb_0
09d24a0e2e8b1126baec2e56d2ac38385b1b2b86e33b9057b23f09d24e88280e   gcr.io/google_containers/pause-amd64:3.0                                                                                   "/pause"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           42 minutes ago      Up 42 minutes                           k8s_POD_etcd-server-ip-172-20-54-17.ec2.internal_kube-system_0db13284a988626a3691585593cee6fa_0
01bf0e1a8259b3566551fa291111fc6eff7be7f8e6b4cf35d20e8a239ca33729   gcr.io/google_containers/pause-amd64:3.0                                                                                   "/pause"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           42 minutes ago      Up 42 minutes                           k8s_POD_kube-scheduler-ip-172-20-54-17.ec2.internal_kube-system_cae1661ea6d72df63c1db473df0cfab8_0
843aa491210d9c36cef53b3ace9a971d9fcca0179853e4506b1862e51a36637a   gcr.io/google_containers/pause-amd64:3.0                                                                                   "/pause"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           42 minutes ago      Up 42 minutes                           k8s_POD_kube-apiserver-ip-172-20-54-17.ec2.internal_kube-system_94d22b056814f1dd76157ab3970e3bd3_0
dd9bd5fbd9cb9d537b16205884fe992a6eecccdfd4c5110204b7c3a87c17816a   gcr.io/google_containers/pause-amd64:3.0                                                                                   "/pause"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           42 minutes ago      Up 42 minutes                           k8s_POD_kube-controller-manager-ip-172-20-54-17.ec2.internal_kube-system_13b92a993591a89c5e20f9facf926675_0
3ff9816e10c63f71da516aa931130848fd1ce7b4b75f4e71a872b8e12e27b821   protokube:1.8.0                                                                                                            "/usr/bin/protokube --channels=s3://cluther-kops-k8s-local-state-store/cluther-kops.k8s.local/addons/bootstrap-channel.yaml --cloud=aws --containerized=true --dns-internal-suffix=internal.cluther-kops.k8s.local --dns=gossip --etcd-image=gcr.io/google_containers/etcd:2.2.1 --initialize-rbac=true --master=true --v=4"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       42 minutes ago      Up 42 minutes                           cranky_minsky
__SPLIT__
14 19 0:14 / /sys rw,nosuid,nodev,noexec,relatime shared:7 - sysfs sysfs rw
15 19 0:3 / /proc rw,nosuid,nodev,noexec,relatime shared:12 - proc proc rw
16 19 0:5 / /dev rw,relatime shared:2 - devtmpfs udev rw,size=10240k,nr_inodes=1022441,mode=755
17 16 0:11 / /dev/pts rw,nosuid,noexec,relatime shared:3 - devpts devpts rw,gid=5,mode=620,ptmxmode=000
18 19 0:15 / /run rw,nosuid,relatime shared:5 - tmpfs tmpfs rw,size=1639580k,mode=755
19 0 8:1 / / rw,relatime shared:1 - ext4 /dev/sda1 rw,errors=remount-ro,data=ordered
20 14 0:16 / /sys/kernel/security rw,nosuid,nodev,noexec,relatime shared:8 - securityfs securityfs rw
21 16 0:17 / /dev/shm rw,nosuid,nodev shared:4 - tmpfs tmpfs rw
22 18 0:18 / /run/lock rw,nosuid,nodev,noexec,relatime shared:6 - tmpfs tmpfs rw,size=5120k
23 14 0:19 / /sys/fs/cgroup ro,nosuid,nodev,noexec shared:9 - tmpfs tmpfs ro,mode=755
24 23 0:20 / /sys/fs/cgroup/systemd rw,nosuid,nodev,noexec,relatime shared:10 - cgroup cgroup rw,xattr,release_agent=/lib/systemd/systemd-cgroups-agent,name=systemd
25 14 0:21 / /sys/fs/pstore rw,nosuid,nodev,noexec,relatime shared:11 - pstore pstore rw
26 23 0:22 / /sys/fs/cgroup/cpuset rw,nosuid,nodev,noexec,relatime shared:13 - cgroup cgroup rw,cpuset
27 23 0:23 / /sys/fs/cgroup/cpu,cpuacct rw,nosuid,nodev,noexec,relatime shared:14 - cgroup cgroup rw,cpu,cpuacct
28 23 0:24 / /sys/fs/cgroup/devices rw,nosuid,nodev,noexec,relatime shared:15 - cgroup cgroup rw,devices
29 23 0:25 / /sys/fs/cgroup/freezer rw,nosuid,nodev,noexec,relatime shared:16 - cgroup cgroup rw,freezer
30 23 0:26 / /sys/fs/cgroup/net_cls,net_prio rw,nosuid,nodev,noexec,relatime shared:17 - cgroup cgroup rw,net_cls,net_prio
31 23 0:27 / /sys/fs/cgroup/blkio rw,nosuid,nodev,noexec,relatime shared:18 - cgroup cgroup rw,blkio
32 23 0:28 / /sys/fs/cgroup/perf_event rw,nosuid,nodev,noexec,relatime shared:19 - cgroup cgroup rw,perf_event
33 15 0:29 / /proc/sys/fs/binfmt_misc rw,relatime shared:20 - autofs systemd-1 rw,fd=23,pgrp=1,timeout=300,minproto=5,maxproto=5,direct
34 16 0:13 / /dev/mqueue rw,relatime shared:21 - mqueue mqueue rw
35 14 0:6 / /sys/kernel/debug rw,relatime shared:22 - debugfs debugfs rw
36 16 0:30 / /dev/hugepages rw,relatime shared:23 - hugetlbfs hugetlbfs rw
37 14 0:31 / /sys/fs/fuse/connections rw,relatime shared:24 - fusectl fusectl rw
39 18 0:33 / /run/rpc_pipefs rw,relatime shared:25 - rpc_pipefs rpc_pipefs rw
68 18 0:34 / /run/user/118 rw,nosuid,nodev,relatime shared:53 - tmpfs tmpfs rw,size=819792k,mode=700,uid=118,gid=125
170 18 0:36 / /run/user/0 rw,nosuid,nodev,relatime shared:152 - tmpfs tmpfs rw,size=819792k,mode=700
175 33 0:37 / /proc/sys/fs/binfmt_misc rw,relatime shared:157 - binfmt_misc binfmt_misc rw
