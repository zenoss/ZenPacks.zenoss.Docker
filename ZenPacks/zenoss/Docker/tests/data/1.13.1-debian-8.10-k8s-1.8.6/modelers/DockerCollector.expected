# vim: ft=yaml
command: "docker -v 2>/dev/null ; echo __SPLIT__ ; sudo docker ps -a --no-trunc 2>/dev/null ; echo __SPLIT__ ; cat /proc/self/mountinfo 2>/dev/null"

expected:
  - docker_version: "Docker version 1.13.1, build 092cba3"

  - relname: "docker_containers"
    maps:
      - id: "4f2e3c742f4f1d54fbe3eb86d378627a96c59bd05c1f8c4f867eb2060be0171f"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_dns-controller_dns-controller-5cbcd846f9-r8km7_kube-system_c568bedb-0d13-11e8-b83e-0267931fa8b6_0"
        image: "kope/dns-controller@sha256:97f80ad43ff833b254907a0341c7fe34748e007515004cf0da09727c5442f53b"
        command: "\"/usr/bin/dns-controller --watch-ingress=false --dns=gossip --gossip-seed=127.0.0.1:3999 --zone=*/* -v=2\""
        created: "40 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "91bd7d905685984cb986156636c87c2cc45bf75a2fa4a341e6f876ffae154a68"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_POD_dns-controller-5cbcd846f9-r8km7_kube-system_c568bedb-0d13-11e8-b83e-0267931fa8b6_0"
        image: "gcr.io/google_containers/pause-amd64:3.0"
        command: "\"/pause\""
        created: "41 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "ce56193cdac7a26e9ec80e8099b1033819c50705627a1247672eb64a4f4bc042"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_etcd-container_etcd-server-events-ip-172-20-54-17.ec2.internal_kube-system_d291a7c4584f864ad771c279f501226b_0"
        image: "gcr.io/google_containers/etcd@sha256:19544a655157fb089b62d4dac02bbd095f82ca245dd5e31dd1684d175b109947"
        command: "\"/bin/sh -c '/usr/local/bin/etcd 2>&1 | /bin/tee -a /var/log/etcd.log'\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "3aa35b7f84789c681444430fb22cf0591b126b610774a84c828f6b632d5ae806"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_kube-proxy_kube-proxy-ip-172-20-54-17.ec2.internal_kube-system_be7ce701e54caaf75dcc19fcdc7eb5bb_0"
        image: "gcr.io/google_containers/kube-proxy@sha256:4dc8aefa2bfe010b5cd820de67e03493a37489696f71339da604de47815ec00a"
        command: "\"/bin/sh -c '/usr/local/bin/kube-proxy --cluster-cidr=100.96.0.0/11 --conntrack-max-per-core=131072 --hostname-override=ip-172-20-54-17.ec2.internal --kubeconfig=/var/lib/kube-proxy/kubeconfig --master=https://127.0.0.1 --oom-score-adj=-998 --resource-container=\\\"\\\" --v=2 2>&1 | /usr/bin/tee -a /var/log/kube-proxy.log'\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "ae1b68661a64df089501c1f455eec44c07ae891d5272d02c98fe63bec9a1475f"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_etcd-container_etcd-server-ip-172-20-54-17.ec2.internal_kube-system_0db13284a988626a3691585593cee6fa_0"
        image: "gcr.io/google_containers/etcd@sha256:19544a655157fb089b62d4dac02bbd095f82ca245dd5e31dd1684d175b109947"
        command: "\"/bin/sh -c '/usr/local/bin/etcd 2>&1 | /bin/tee -a /var/log/etcd.log'\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "c699903676052404e0ba754eb7f43c21e36b6833568bbd0f42191ba4ca8bcb05"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_kube-scheduler_kube-scheduler-ip-172-20-54-17.ec2.internal_kube-system_cae1661ea6d72df63c1db473df0cfab8_0"
        image: "gcr.io/google_containers/kube-scheduler@sha256:4dcf7b2872bd9086ce236535bf03d2077252aebedbcad5de324042edd428f478"
        command: "\"/bin/sh -c '/usr/local/bin/kube-scheduler --kubeconfig=/var/lib/kube-scheduler/kubeconfig --leader-elect=true --v=2 2>&1 | /bin/tee -a /var/log/kube-scheduler.log'\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "cff2a01ebe258d4a77bb2f037ec6a323f5518250e937797df2190bf42ed6d53d"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_kube-apiserver_kube-apiserver-ip-172-20-54-17.ec2.internal_kube-system_94d22b056814f1dd76157ab3970e3bd3_0"
        image: "gcr.io/google_containers/kube-apiserver@sha256:f0658f0c3344abaae3217d6e08612ea8816a0cdeea1fdd48776dd55d07f17d30"
        command: "\"/bin/sh -c '/usr/local/bin/kube-apiserver --address=127.0.0.1 --admission-control=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota --allow-privileged=true --anonymous-auth=false --apiserver-count=1 --authorization-mode=AlwaysAllow --basic-auth-file=/srv/kubernetes/basic_auth.csv --client-ca-file=/srv/kubernetes/ca.crt --cloud-provider=aws --etcd-servers-overrides=/events#http://127.0.0.1:4002 --etcd-servers=http://127.0.0.1:4001 --insecure-port=8080 --kubelet-preferred-address-types=InternalIP,Hostname,ExternalIP --proxy-client-cert-file=/srv/kubernetes/apiserver-aggregator.cert --proxy-client-key-file=/srv/kubernetes/apiserver-aggregator.key --requestheader-allowed-names=aggregator --requestheader-client-ca-file=/srv/kubernetes/apiserver-aggregator-ca.cert --requestheader-extra-headers-prefix=X-Remote-Extra- --requestheader-group-headers=X-Remote-Group --requestheader-username-headers=X-Remote-User --secure-port=443 --service-cluster-ip-range=100.64.0.0/13 --storage-backend=etcd2 --tls-cert-file=/srv/kubernetes/server.cert --tls-private-key-file=/srv/kubernetes/server.key --token-auth-file=/srv/kubernetes/known_tokens.csv --v=2 2>&1 | /bin/tee -a /var/log/kube-apiserver.log'\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "9d2919d80e851414e1fb63f395e2b91b7a5a97117122a819d9d25bdcfdd8e77a"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_kube-controller-manager_kube-controller-manager-ip-172-20-54-17.ec2.internal_kube-system_13b92a993591a89c5e20f9facf926675_0"
        image: "gcr.io/google_containers/kube-controller-manager@sha256:f06b0a323511a8d5a0c47c4229c3580390a5232761eade11bbe9356739356514"
        command: "\"/bin/sh -c '/usr/local/bin/kube-controller-manager --allocate-node-cidrs=true --attach-detach-reconcile-sync-period=1m0s --cloud-provider=aws --cluster-cidr=100.96.0.0/11 --cluster-name=cluther-kops.k8s.local --cluster-signing-cert-file=/srv/kubernetes/ca.crt --cluster-signing-key-file=/srv/kubernetes/ca.key --configure-cloud-routes=true --kubeconfig=/var/lib/kube-controller-manager/kubeconfig --leader-elect=true --root-ca-file=/srv/kubernetes/ca.crt --service-account-private-key-file=/srv/kubernetes/server.key --use-service-account-credentials=true --v=2 2>&1 | /bin/tee -a /var/log/kube-controller-manager.log'\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "bc96cba9431214475a6df4449a92cb4fc6597f562e42f1747973881bd7d24c4f"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_POD_etcd-server-events-ip-172-20-54-17.ec2.internal_kube-system_d291a7c4584f864ad771c279f501226b_0"
        image: "gcr.io/google_containers/pause-amd64:3.0"
        command: "\"/pause\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "9243cbcb485b8a22b06f728512757109e7f56e664b5e3829f9e2320083480b55"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_POD_kube-proxy-ip-172-20-54-17.ec2.internal_kube-system_be7ce701e54caaf75dcc19fcdc7eb5bb_0"
        image: "gcr.io/google_containers/pause-amd64:3.0"
        command: "\"/pause\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "09d24a0e2e8b1126baec2e56d2ac38385b1b2b86e33b9057b23f09d24e88280e"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_POD_etcd-server-ip-172-20-54-17.ec2.internal_kube-system_0db13284a988626a3691585593cee6fa_0"
        image: "gcr.io/google_containers/pause-amd64:3.0"
        command: "\"/pause\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "01bf0e1a8259b3566551fa291111fc6eff7be7f8e6b4cf35d20e8a239ca33729"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_POD_kube-scheduler-ip-172-20-54-17.ec2.internal_kube-system_cae1661ea6d72df63c1db473df0cfab8_0"
        image: "gcr.io/google_containers/pause-amd64:3.0"
        command: "\"/pause\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "843aa491210d9c36cef53b3ace9a971d9fcca0179853e4506b1862e51a36637a"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_POD_kube-apiserver-ip-172-20-54-17.ec2.internal_kube-system_94d22b056814f1dd76157ab3970e3bd3_0"
        image: "gcr.io/google_containers/pause-amd64:3.0"
        command: "\"/pause\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "dd9bd5fbd9cb9d537b16205884fe992a6eecccdfd4c5110204b7c3a87c17816a"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "k8s_POD_kube-controller-manager-ip-172-20-54-17.ec2.internal_kube-system_13b92a993591a89c5e20f9facf926675_0"
        image: "gcr.io/google_containers/pause-amd64:3.0"
        command: "\"/pause\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "3ff9816e10c63f71da516aa931130848fd1ce7b4b75f4e71a872b8e12e27b821"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "cranky_minsky"
        image: "protokube:1.8.0"
        command: "\"/usr/bin/protokube --channels=s3://cluther-kops-k8s-local-state-store/cluther-kops.k8s.local/addons/bootstrap-channel.yaml --cloud=aws --containerized=true --dns-internal-suffix=internal.cluther-kops.k8s.local --dns=gossip --etcd-image=gcr.io/google_containers/etcd:2.2.1 --initialize-rbac=true --master=true --v=4\""
        created: "42 minutes ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"
